assembl/static2/js/app/actions/debateActions.js

```diff
@@ -33,7 +33,7 @@ export const fetchDebateData = (debateId) => {
       dispatch(resolvedFetchDebateData(debateData));
     }).catch((error) => {
       const firstError = error[0];
-      if (firstError.status === 401) {
+      if (firstError && 'status' in firstError && firstError.status === 401) {
```

Comment:
Oh? Do you know which API from backend caused a crash? Because APIs that relate to V2 should respond with a `JSONError` now, which would ensure that this object would always exist.